app/src/main/res/drawable/back_ic.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
ic_back

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6